### PR TITLE
ENH: Continuing adding contents

### PR DESCRIPTION
**Pull request by oesteban**
I've tried to rely less on text and add more figures. Will continue to work on this along the day, but I send the PR to get additional feedback from @effigies, @mgxd, and @arokem (or anyone who wants to chime in).

### CODE REVIEW

**Comment by oesteban** [comment]
BTW, I've moved the presentation of the grant to the very beginning, and right after the involved teams.

In my mind, you would say at the outskirts that we are developing NiPreps as a consequence of the lessons we learned with fMRIPrep. Then, I like the anti-pattern story to justify it as the most prominent problem, but there are others.

I add further content later.

**Comment by poldrack** [comment]
Thanks! I don't love the reordering - I always like to start my talks with a motivating story or example, which I think the anti-pattern story serves as in this case.  so i'd prefer to keep it in the beginning...

**Comment by oesteban** [comment]
Okay, I'll roll the order back. You're the presenter :D

File: talk/talk.qmd [hunk 1]
```diff
@@ -272,20 +202,40 @@ In addition to maintenance across the board and infrastructure (e.g., website),
...
+::: {.column width="50%"}
+- Overhaul started early 2021 ([Esteban et al., OHBM 2021](https://doi.org/10.31219/osf.io/gy8nt))
+- Faced many technical challenges → numerous bugfixes
+    - vast amount of "edge" cases
+- Opacity of the problem itself → educational materials
+    - Jupyter notebooks
+:::
+
+::: {.column width="50%"}
+![](images/sdcflows-notebook.png)
```
Comment (effigies) [inline talk/talk.qmd:235]: Corrupt file

Comment (oesteban) [reply talk/talk.qmd:235]: done, it renders for me on GH

**Comment by effigies** [comment]
Rendering locally, we've got a few problems. Fix here:  b9f90af

Screenshot from 2022-11-15 11-31-58
Screenshot from 2022-11-15 11-32-10

**Comment by effigies** [comment]
Fixed renders:

Screenshot from 2022-11-15 11-40-33
Screenshot from 2022-11-15 11-40-39

**Comment by oesteban** [comment]
Okay @poldrack, I think this should make for a good base for you to pick what you like the best.

**Comment by poldrack** [comment]
thanks!

**Pull request merged**